ctapipe/utils/tests/test_astro.py

```diff
@@ -0,0 +1,20 @@
+from ..astro import get_bright_stars
+from astropy.coordinates import SkyCoord
+from astropy import units as u
+
+try:
+    import ctapipe_resources
+except ImportError:
+    raise RuntimeError("Please install the 'ctapipe-extra' package, "
+                       "which contains the ctapipe_resources module "
+                       "needed by ctapipe. (conda install ctapipe-extra)")
```

Comment:
This should not be needed. A similar error is raised by `get_dataset_path` if the file is not found.

Reply:
yes, sorry, I suspected. Was just a "test" to figure out why the test by Travis CI did not succeed. Just removing it again.